### PR TITLE
Citrix ADC Data Connector instructions and Parser Changes

### PR DESCRIPTION
**Pull request by v-mchatla**
Citrix ADC Data Connector and Parser Changes

   Required items, please complete
   
   Change(s):
   - Citrix ADC supports only syslog format

   Reason for Change(s):
   - Existing connector and parser pointing to CommonSecurityLog Table

   Version Updated:
   - Yes 3.0.0

   Testing Completed:
   - Yes

### CODE REVIEW

**Comment by v-mchatla** [comment]
Hi @devikamehra 
As suggested, I have added sample data. Can you please review the changes and share your feedback.
Thanks

File: Solutions/Citrix ADC/Data Connectors/Connector_CitrixADC_syslog.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            "description": "Open Log Analytics to check if the logs are received using the Syslog schema.\n\n>**NOTE:** It may take up to 15 minutes before new logs will appear in Syslog table.",
+            "instructions": []    
+        }		
+    ],
+    "metadata" : {
+        "id": "c546d5dd-dfec-41fb-a680-af0c66531adc",
+        "version": "1.0.1",
+        "kind": "dataConnector",
+        "source": {
+            "kind": "community"
```
Comment (devikamehra) [inline Solutions/Citrix ADC/Data Connectors/Connector_CitrixADC_syslog.json:124]: Shouldn't this be Microsoft?

**Review by devikamehra — DISMISSED** [review]
Changes in Parser looks good to me. Please check the data connector and solution deployment before merging.

File: Solutions/Citrix ADC/Data Connectors/Connector_CitrixADC_syslog.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        {
+            "name": "Syslog",
+            "lastDataReceivedQuery": "CitrixADCEvent\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "CitrixADCEvent\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
```
Comment (anki-narravula) [inline Solutions/Citrix ADC/Data Connectors/Connector_CitrixADC_syslog.json:30]: This query can be optimized like 
CitrixADCEvent\n   | where  TimeGenerated > ago(3d) \n| take 1 \n   | project IsConnected = true

Comment (v-mchatla) [reply Solutions/Citrix ADC/Data Connectors/Connector_CitrixADC_syslog.json:30]: sure, will make the requested change and update.
Thanks

**Review by anki-narravula — APPROVED** [review]

**Pull request merged**